app.py

```diff
@@ -119,24 +181,25 @@ def index():
 
 
 @app.route("/post/<slug>")
-def post(slug):
+def post(slug: str):
+    """Renders a view for an specific post (defined by the slug)"""
```

Comment:
The **view** (a function that builds the response to a HTTP request) renders the *template* for a specific post

Reply:
(also update in the other views)